.storybook/stories/KvCarousel.stories.js

```diff
@@ -10,16 +10,16 @@ Vue.use(Vue2TouchEvents);
 
 const defaultCarouselSlides = `
 	<template slot="default" slot-scope="props">
-		<kv-carousel-slide :transition-name="props.transitionName">
+		<kv-carousel-slide :transition-name="props.transitionName" style="background-color: green;">
```

Comment:
Keeping the green background color here as a way to illustrate the slide dimensions